sentry_dart_plugin/lib/src/configuration.dart

```diff
@@ -0,0 +1,155 @@
+import 'dart:io';
+
+import 'package:package_config/package_config.dart';
+import 'package:system_info/system_info.dart';
+import 'package:yaml/yaml.dart';
+
+import 'utils/extensions.dart';
+import 'utils/log.dart';
+
+class Configuration {
+  // cannot use ${Directory.current.path}/build since --split-debug-info allows
+  // setting a custom path which is a sibling of build
+  String buildFilesFolder = Directory.current.path;
+
+  late bool uploadNativeSymbols;
+  late bool uploadSourceMaps;
+  late bool includeNativeSources;
+  late bool wait;
+  late String? project;
+  late String? org;
+  late String? authToken;
+  late String? logLevel;
+  String? _assetsPath;
+  late String? cliPath;
+  final String _fileSeparator = Platform.pathSeparator;
+  late String version;
+  late String name;
+  late String webBuildFilesFolder;
+
+  dynamic _getPubspec() {
+    final pubspecString = File("pubspec.yaml").readAsStringSync();
+    final pubspec = loadYaml(pubspecString);
+    return pubspec;
+  }
+
+  Future<void> getConfigValues(List<String> arguments) async {
+    const taskName = 'reading config values';
+    Log.startingTask(taskName);
+
+    await _getAssetsFolderPath();
+    _findCliPath();
+    final pubspec = _getPubspec();
+    final config = pubspec['sentry_plugin'];
+
+    version = config?['release']?.toString() ?? pubspec['version'].toString();
+    name = pubspec['name'].toString();
+
+    uploadNativeSymbols = config?['upload_native_symbols'] ?? true;
+    uploadSourceMaps = config?['upload_source_maps'] ?? false;
+    includeNativeSources = config?['include_native_sources'] ?? false;
+
+    // uploading JS and Map files need to have the correct folder structure
+    // otherwise symbolication fails, the default path for the web build folder is build/web
+    // but can be customized so making it flexible.
+    final webBuildPath = config?['web_build_path']?.toString() ?? 'build/web';
+    webBuildFilesFolder = '$buildFilesFolder$_fileSeparator$webBuildPath';
+
+    // pro tip: add rule: stack.abs_path:org-dartlang-sdk* -app
```

Comment:
I will try to make it a global config for Sentry, will talk next week with @untitaker

Reply:
Whatd oes it do?

Reply:
that means every frame that contains `org-dartlang-sdk*` won't be considered for grouping issues together.

Reply:
this is a good thing because in-app detection does not work well for the Web since there's minification and we cannot detect it automatically at runtime.